lib/routes/bupt/scss.ts

```diff
@@ -0,0 +1,139 @@
+                    label: '通知公告'
+                }
+            ]
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['scss.bupt.edu.cn/index/xwdt.htm'],
+            target: '/scss/xwdt',
+        },
+        {
+            source: ['scss.bupt.edu.cn/index/tzgg1.htm'],
+            target: '/scss/tzgg',
+        },
+    ],
+    name: '网络空间安全学院',
+    maintainers: ['ziri2004'],
+    handler,
+    url: 'scss.bupt.edu.cn',
+};
+
+async function handler(ctx: Context) {
+    let type = ctx.req.param('type'); // 默认类型为通知公告
+    if (!type) {
+        type = 'tzgg';
+    }
+    const rootUrl = 'https://scss.bupt.edu.cn';
+    let currentUrl;
+    let pageTitle;
+
+    if (type === 'xwdt') {
+        currentUrl = `${rootUrl}/index/xwdt.htm`;
+        pageTitle = '新闻动态';
+    } else if (type === 'tzgg') {
+        currentUrl = `${rootUrl}/index/tzgg1.htm`;
+        pageTitle = '通知公告';
+    } else {
+        throw new Error('Invalid type parameter');
+    }
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = load(response.data);
+
+    const selector = type === 'xwdt' ? '.m-list3 li' : '.Newslist li';
+
+    const list = $(selector)
+        .toArray()
+        .map((item) => {
+            const $item = $(item);
+            const $link = $item.find('a');
+            if ($link.length === 0 || !$link.attr('href')) {
+                return null;
+            }
+            const href = $link.attr('href');
+            const link = new URL(href, rootUrl).href;
+
+            return {
+                title: $link.text().trim(),
+                link,
+            };
+        })
+        .filter(Boolean);
+
+
+    const items = await Promise.all(
+        list.map((item) =>
+            cache.tryGet(item.link, async () => {
+                const detailResponse = await got({ url: item.link });
+                const content = load(detailResponse.data);
+                const newsContent = content('.v_news_content');
+
+                item.description = newsContent.text().trim();
+
+                newsContent.find('p, span, strong').each(function () {
+                    const element = content(this);
+                    const text = element.text().trim();
+
+
+                    if (text === '') {
+                        element.remove();
+                    } else {
+                        element.replaceWith(text);
+                    }
+                });
+
+                // 清理后的内容转换为文本
+                const cleanedDescription = newsContent.text().trim();
+
+                // 提取并格式化发布时间
+                item.description = cleanedDescription;
+                item.pubDate = timezone(parseDate(content('.info').text().replace('发布时间：', '').trim()), +8);
```

Comment:
This selects an invalid date as you can see it from https://github.com/DIYgod/RSSHub/pull/19291#issuecomment-2948194809